### ISSUE TITLE
minikube service

### ISSUE THREAD

**Issue by jadekler** [kubernetes/minikube#715]
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
BUG

**Minikube version** (use `minikube version`):
```
 2016-10-19 07:47:26 ☆  |2.3.0| Jeans-MacBook-Pro in ~/Downloads
○ → minikube version
minikube version: v0.11.0
```

**Environment**:
- **OS** (e.g. from /etc/os-release): OS El Capitan 10.11.6
- **VM Driver** (e.g. `cat ~/.minikube/machines/minikube/config.json | grep DriverName`): Virtualbox
- **Docker version** (e.g. `docker -v`): Docker version 1.12.2, build bb80604
- **Install tools**:
- **Others**:

**What happened**:
Minikube crashed when I typed `minikube service`

**What you expected to happen**:
Syntax or missing argument error

**How to reproduce it** (as minimally and precisely as possible):
```
 2016-10-19 07:46:53 ☆  |2.3.0| Jeans-MacBook-Pro in ~/Downloads
○ → minikube service
Please specify a service name.
========================================An error has occurred.  Would you like to opt in to sending anonymized crash information to minikube to help prevent future errors?(To opt out of these messages, run the command)
	minikube config set WantReportErrorPrompt false========================================
Please enter your response [Y/n]:
```

**Anything else do we need to know**:

**Comment by r2d4** [comment]
This was fixed by the #732.  No longer prompts the user for error reporting for a lot of these kinds of errors

**Comment by jadekler** [comment]
Thanks! @r2d4
